.github/workflows/sycl_nightly.yml

```diff
@@ -10,24 +10,30 @@ on:
       - '.github/workflows/sycl_nightly.yml'
 
 jobs:
+  resolve_matrix:
+    name: Resolve Test Matrix
+    uses: ./.github/workflows/sycl_resolve_test_matrix.yml
+    with:
+      lts_config: "ocl_gen9;ocl_x64"
+
   ubuntu2004_build_test:
     if: github.repository == 'intel/llvm'
     uses: ./.github/workflows/sycl_linux_build_and_test.yml
+    needs: resolve_matrix
```

Comment:
I think this stopped running SYCL end-to-end tests due to missing

```
with:
  lts_matrix: needs.resolve_matrix.outputs.lts_matrix
```

@bader , do we need to fix that? Or should we just remove matrix generation from nightly alltogether?

Reply:
I'll let @sergey-semenov to decide on the runtime library validation requirements/process.

Reply:
@stdale-intel and I want to use all the E2E tests as the criteria for "latest-good" nightly release, so I'm re-enabling missing tasks in https://github.com/intel/llvm/pull/10254.